.github/workflows/deploy.yml

```diff
@@ -0,0 +1,25 @@
+name: Onceler Deploy
+
+on:
+  push:
+    branches: [main]
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby installation
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: false
+
+      - name: Deploy to cloud.gov
+        uses: cloud-gov/cg-cli-tools@main
+        with:
+          cf_api: ${{ secrets.CF_API }}
+          cf_username: ${{ secrets.CF_USERNAME }}
+          cf_password: ${{ secrets.CF_PASSWORD }}
+          cf_org: ${{ secrets.CF_ORG }}
+          cf_space: ${{ secrets.CF_SPACE }}
```

Comment:
Are you keeping the org and space "secret" to keep it generic or are they considered sensitive?

Reply:
I consider them sensitive, this is a public repository after all.